ansible/deploy.yml

```diff
@@ -76,6 +76,21 @@
         when: package is not defined
         tags: [package]
 
+      - name: Download remote package
+        get_url:
+          url: "{{ package }}"
+          dest: "{{ packages_path }}/{{ package|basename }}"
+          checksum: "{{ package_checksum|default(omit) }}"
```

Comment:
I see `default(omit)` is [documented](https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#omitting-parameters).